css/index.css

```diff
@@ -1 +1,515 @@
+border-collapse: collapse;
+border-spacing: 0;
+}
+/* Set every element's box-sizing to border-box */
+* {
+box-sizing: border-box;
+}
+html, body {
+height: 100%;
+font-family: 'Titillium Web', sans-serif;
+}
+h1, h2, h3, h4, h5 {
+font-family: 'Bangers', cursive;
+letter-spacing: 1px;
+margin-bottom: 15px;
+}
+/* Your code starts here! */
+
+html {
+    max-width: 100%;
+    font-size: 62.5%;
+}
+
+.navBar {
+    display: flex;
+    justify-content: center;
+    position: fixed;
+    background: white;
+    max-width: 100%;
+}
+
+.ops {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    background: white;
+    max-width: 100%;
+}
+
+.serv, .prod, .vis, .feat, .abou, .cont {
+    background-color:white;
+    border: none;
+    color: black;
+    font-size: 2rem;
+    transition: 0.3s;
+    font-family: 'Permanent Marker', cursive;
+    text-decoration: none;
+    padding: 20px;
+    padding-top: 25px;
+}
+
+.serv:hover, .prod:hover, .vis:hover, .feat:hover, .abou:hover, .cont:hover {
+    background-color: #0034A3;
+    color: white;
+}
+
+.logoImg {
+    margin-top: 15px;
+    height: 3rem;
+    background-color: white;
+}
+
+body{
+    max-width: 100%;
+}
+
+.frontTitle {
+    display: flex;
+    justify-content: center;
+    font-size: 7rem;
+    max-width: 100%;
+}
+
+.title {
+    margin-top: 175px;
+    display: flex;
+    justify-content: center;
+    max-width: 100%;
+}
+
+.started {
+    display: flex;
+    justify-content: center;
+    font-size: 3rem;
+    background-color: white;
+    box-shadow: 0 5px #666;
+    border: solid black;
+    padding: 10px;
+    max-width: 100%;
+}
+
+.column {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+}
+
+.headerImg{
+    padding-left: 13%;;
+}
+
+.topDesc {
+    display: flex;
+    justify-content: space-evenly;
+    margin-top: 2rem;
+    border-top: 5px solid black;
+    width: 100%;
+}
+
+.features, .about{
+    margin-top: 25px;
+    display: flex;
+    flex-direction: column;
+    align-content: flex-start;
+    width: 100%;
+}
+
+p {
+    width: 80%;
+    font-size: 2rem;
+}
+.middle-img {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+}
+
+.started:hover {
+    background-color: lightgrey;
+}
+
+.bottom {
+    margin-top: 25px;
+    display: flex;
+    justify-content: space-evenly;
+    border-bottom: 5px solid black;
+    padding-bottom: 25px;
+    width: 100%;
+}
+
+h3 {
+    font-size: 3rem;
+}
+
+.smallP {
+width: 100%;;
+}
+
+.contact {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    font-size: 2rem;
+    margin-top: 30px;
+    margin-left: 0;
+}
+
+.number {
+    margin-top: 2rem;
+    margin-bottom: 2rem;
+}
+
+.copyright {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-bottom: 10rem;
+    margin-top: 5rem;
+}
+.container {
+    font-size: 62.5%;
+    margin: 0 auto;
+    max-width: 880px;
+}
+
+
+@media(max-width: 800px) {
```

Comment:
You could make your code more dry by only adding the lines that changed to your media query.